app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :micropost
+  belongs_to :user
+  validates :micropost_id, presence: true
+  validates :user_id, presence: true
+  validates :content, presence: true, length: { maximum: 140}  
```

Comment:
【細かい】スペースいれる／いれないを統一したい

```suggestion
  validates :content, presence: true, length: { maximum: 140 }
```